src/locks/locks/events/hit-item-frame.ts

```diff
@@ -0,0 +1,46 @@
+import { Location } from 'org.bukkit';
+import { ItemFrame } from 'org.bukkit.entity';
+import { PlayerInteractEntityEvent } from 'org.bukkit.event.player';
+import { dataType } from '../../../common/datas/holder';
+import { dataView } from '../../../common/datas/view';
+import { CUSTOM_DATA_KEY } from '../../../common/items/CustomItem';
+import { VkItem } from '../../../common/items/VkItem';
+import { isLockableMaterial } from '../blocklocks/block-lock-list';
+import { createLockItem, LOCK_DATA } from '../lock-items';
+const lockDataType = dataType(CUSTOM_DATA_KEY, LOCK_DATA);
+
+/**
+ * Allow players to destroy floating locks
```

Comment:
What could cause floating locks? Is this just to make potential bugs impact players less?